README.md

```diff
@@ -6,7 +6,7 @@ To build and install the snap:
 ```
  $ snapcraft
  ...
- $ snap install --devmode --dangerous plexmediaserver_*_amd64.snap
+ $ sudo snap install --devmode --force-dangerous plexmediaserver_*_amd64.snap
```

Comment:
With snapd 2.15 this is actually just --dangerous. Not sure if --force-dangerous is then still supported. 2.15 should have landed everywhere already.